.circleci/config.yml

```diff
@@ -276,10 +276,6 @@ jobs:
           name: Kick off Cordova automatic dependency update
           command: bundle exec fastlane bump_hybrid_dependencies repo_name:cordova-plugin-purchases
           when: always
-      - run:
-          name: Kick off Unity automatic dependency update
-          command: bundle exec fastlane bump_hybrid_dependencies repo_name:purchases-unity
-          when: always
```

Comment:
Within Unity, we release 2 versions of the SDK, one with the billing client and one without. We could potentially release the version with the billing client (the one different is the one without it, since it's meant for devs using Unity's billing client SDK instead of our own). But just not bumping unity is probably safer, until we fully support Unity 👍